.github/workflows/codeql-analysis.yml

```diff
@@ -1,24 +1,11 @@
-# For most projects, this workflow file will not need changing; you simply need
-# to commit it to your repository.
-#
-# You may wish to alter this file to override the set of languages analyzed,
-# or to provide custom queries or build logic.
-#
-# ******** NOTE ********
-# We have attempted to detect the languages in your repository. Please check
-# the `language` matrix defined below to confirm you have the correct set of
-# supported CodeQL languages.
-#
 name: "CodeQL"
 
 on:
   push:
-    branches: [ main ]
+    branches: [ main, release-2.3, release-2.2, release-2.1, release-2.0, release-1.4 ]
```

Comment:
I think I'd like to only see LTS branches here:
```
branches: [ main, release-2.2, release-1.4 ]
```
Remediating and keeping track of all the branches may be a lot of effort. And since all future LTS branches are created from `main` future branches should only have new issues for changes not introduced from cherry-picks.